src/store/actions/searching-project-user/project/project.js

```diff
@@ -93,23 +93,29 @@ export const _onSubmitFormAnswers = (answers, organizationId, projectId) => {
       uid: studentUID,
     } = student;
 
+    console.log("getState",getState)
     console.log('submitting form answers')
+    console.log("uid",student.uid)
+    console.log("studentId",studentUID)
+    console.log("studentEmail",student.studentEmail)
+    console.log("orgId",orgId)
     try {
       await firebase_db
         .collection("organization")
-        .doc(`${organizationId}`)
+        .doc(`${orgId}`)
         .collection("projects")
         .doc(`${projectId}`)
         .collection("formSubmission")
-        .doc(`${studentUID}`)
+        // .doc(`${studentUID}`)
+        .doc()
         .set(
           {
-            answers: answers,
-            studentInfo: {
-                studentName: studentName,
-                studentEmail: studentEmail,
-                studentUID: studentUID 
-            }
+            answers: answerInput,
+            // studentInfo: {
+            //     studentName: studentName,
+            //     studentEmail: studentEmail,
+            //     studentUID: studentUID 
+            // }
```

Comment:
Can't see the reason why this was deleted. @TrNgTien

Reply:
Plz check your messenger i told u the detail in messenger
.doc(${studentUID}) is the problem why we can push the data of answers to Firebase.Becz the getState() can not get the State from reducer so it was wrong